ci/deps/actions-310.yaml

```diff
@@ -49,8 +49,10 @@ dependencies:
   - scipy>=1.7.1
   - sqlalchemy>=1.4.16
   - tabulate>=0.8.9
-  - tzdata>=2022a
   - xarray>=0.21.0
   - xlrd>=2.0.1
   - xlsxwriter>=1.4.3
   - zstandard>=0.15.2
+
+  - pip:
+    - tzdata>=2022.1; platform_system=="Windows"
```

Comment:
It's probably better to be consistent and go with the pip installed tzdata, so we don't get surprises if/when Github updates its runner images.

I guess there's a risk of things breaking for people without the tzdata package installed, but that's more of a zoneinfo/Python stdlib problem.

Reply:
agree, but isn't that what I've done (i.e. pip-installed tzdata)?

Reply:
I meant on all platforms not just Windows, sorry if I was unclear.

Reply:
ok, sure, I've removed the windows-specific condition, and have installed/required it everywhere